controllers/front/ProductController.php

```diff
@@ -1130,9 +1130,10 @@ public function getTemplateVarProduct()
         $product_full['quantity_label'] = ($this->product->quantity > 1) ? $this->trans('Items', [], 'Shop.Theme.Catalog') : $this->trans('Item', [], 'Shop.Theme.Catalog');
         $product_full['quantity_discounts'] = $this->quantity_discounts;
 
-        if ($product_full['unit_price_ratio'] > 0) {
-            $unitPrice = ($productSettings->include_taxes) ? $product_full['price'] : $product_full['price_tax_exc'];
-            $product_full['unit_price'] = $unitPrice / $product_full['unit_price_ratio'];
+        $product_full['unit_price_ratio'] = $product_full['unit_price'] != 0 ? ($product_full['price_tax_exc'] / $product_full['unit_price']) : 0;
```

Comment:
Is it `price_tax_exc` and not `price_tax_excl` ?

Reply:
I copied the previous code which used `price_tax_exc` (see the previous code on the left) But I'm gonna check just in case

Reply:
I don't know if it was a typo at the time, but anyway `getProductProperties` does add this field:
https://github.com/PrestaShop/PrestaShop/blob/1341e8e45856da3f6922f7a3edb4cd3081045a7c/classes/Product.php#L5645